types/puppet/metadata/os_support.pp

```diff
@@ -1,5 +1,5 @@
 # The 'operating_support' data structure in metadata.json
 type Simplib::Puppet::Metadata::OS_support = Struct[{
   'operatingsystem'        => String,
-  'operatingsystemrelease' => Array[String]
+  'operatingsystemrelease' => Optional[Array[String]]
```

Comment:
I don't know why you made this change.  The original changelog entry didn't make sense to me so I updated it to indicate the type change.